cmd/jujud/agent/machine/manifolds.go

```diff
@@ -432,15 +432,15 @@ func commonManifolds(config ManifoldsConfig) dependency.Manifolds {
 		// The modelcache manifold creates a cache.Controller and keeps
 		// it up to date using an all model watcher. The controller is then
 		// used by the apiserver.
-		modelCacheName: ifDatabaseUpgradeComplete(ifController(modelcache.Manifold(modelcache.ManifoldConfig{
+		modelCacheName: ifDatabaseUpgradeComplete(modelcache.Manifold(modelcache.ManifoldConfig{
```

Comment:
certainly I wouldn't expect all machine agents to be running the model cache

Reply:
`ifDatabaseUpgradeComplete` already implies this.